.github/workflows/ci.yml

```diff
@@ -116,7 +116,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node-version: [14, 16, 18]
+        node-version: [16, 18, 20]
```

Comment:
@n1ru4l how does it pick the `16` there? because pnpm v8 needs at least `16.14` to run?

So I'm curious if we need to specify that precisely 🤔

Reply:
latest 16 i assume

Reply:
ok, good then!